### ISSUE TITLE
Remove numbers: Asset Tab

### ISSUE THREAD

**Issue by 0x4Graham** [centrifuge/apps#2314]
In the assets tab, 

1. remove the numbers associated with the asset (1,2,3,4). 
2. Sort the list by descending maturity date.
3. Set amount to 0 if the asset is empty
4. Make the asset name column wider so we can see the full asset name, or full text of the asset. We will need this for DYF

**Comment by kattylucy** [comment]
for number 1 do you mean the number here?



![Image](https://github.com/user-attachments/assets/946a3768-aaec-4255-be3d-32344ecc7a54)

**Comment by 0x4Graham** [comment]
Yup, let's remove those numbers (they end up being weirdly ordered) 

cc @sonam-jo
